CMakeLists.txt

```diff
@@ -11,11 +11,18 @@ find_package(rclcpp REQUIRED)
 find_package(rclcpp_action REQUIRED)
 find_package(Threads REQUIRED)
 
+include_directories(include)
```

Comment:
Using the global function `include_directories` is considered bad practice in modern cmake. Please use `target_include_directories` and export the library as part of the install set.

Reply:
done